libs/tree-sitter-wing/grammar.js

```diff
@@ -463,9 +461,6 @@ module.exports = grammar({
         )
       ),
 
-    compiler_dbg_panic: ($) => "😱",
```

Comment:
If I recall the whole idea of this "feature" (or backdoor) was to test that panics are handled properly and diagnostics are reported for them.

Maybe reconsider removing them.

Reply:
Could be interesting to instead elevate this past a debug feature and turn it into an intrinsic ala `panic!` or `todo!`

Reply:
Noted.

(I think there are better ways to test the custom panic handler / CLI error message that don't require inserting special syntax and plumbing it through all of the stages of the compiler)